gcore/src/exec.rs

```diff
@@ -147,3 +149,9 @@ pub fn wake(waker_id: MessageId) {
         sys::gr_wake(waker_id.as_slice().as_ptr());
     }
 }
+
+pub fn actor_id() -> ActorId {
```

Comment:
Two concerns:
1. The function name seems to be irrelevant to its purpose. It always returns the program ID so `actor_id` looks too general and confusing.
2. This function needs to be documented.

Reply:
yes , but I have heard that program_id will be deprecated soon. Also `ActorId` struct is available to be used here, `ProgramId` is not.

Reply:
You are right about *types* `ActorId` vs `ProgramId`. But the *function name* should reflect that it does. When I see `exec::actor_id()` in code, it is not obvious to me, who is this actor. So in my opinion `exec::program_id()` or `exec::my_id()` or even `exec::me()` is more speaking for the end user. I'd choose `exec::program_id()` as the most neutral name.